lib/scala/project-manager/src/main/scala/org/enso/projectmanager/service/MonadicProjectValidator.scala

```diff
@@ -1,38 +0,0 @@
-package org.enso.projectmanager.service
-
-import cats.MonadError
-import cats.implicits._
-import org.enso.pkg.validation.{InvalidNameError, NameValidation}
-
-/** MTL implementation of the project validator. */
-class MonadicProjectValidator[F[_, _]](implicit
```

Comment:
_"Monadic(!) project validator"_ - what a scary name. I am glad a file like this could be deleted.

Reply:
Don't fear the monad